app/operations/base_operations/activity_recording.rb

```diff
@@ -1,23 +1,25 @@
 module BaseOperations
   module ActivityRecording
-    def fetch_project
-      case model
+    def fetch_project(record)
```

Comment:
Why this change?

Reply:
Because I'm receiving a collection of stories over the model, so I iterated with the model in line 15 passing the record as a story!